cli/tauri.js/test/jest/__tests__/build.spec.js

```diff
@@ -43,35 +43,37 @@ describe('Tauri Build', () => {
     distDir: distDir
   }
 
-  for (const debug of [true, false]) {
-    it(`works with the embedded-server ${debug ? 'debug' : 'release'} mode`, () => {
+  test.each([['debug'], ['release']])(
+    'works with the embedded-server $a mode',
+    mode => {
       return runBuildTest({
         build,
         ctx: {
-          debug
+          debug: mode === 'debug'
         },
         tauri: {
           embeddedServer: {
             active: true
           }
         }
       })
-    })
-  }
+    }
+  )
 
-  for (const debug of [true, false]) {
-    it(`works with the no-server ${debug ? 'debug' : 'release'} mode`, () => {
+  test.each([['debug'], ['release']])(
```

Comment:
You can use `it.each` instead, that would stay in line with the rest of the tests.